configsamples/bgppeer_with_bfdprofile.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: metallb.io/v1beta1
+kind: BFDProfile
+metadata:
+  name: testbfdprofile
+  namespace: metallb-system
+spec:
+  receiveInterval: 380
+  transmitInterval: 270
+---
+apiVersion: metallb.io/v1beta2
+kind: BGPPeer
+metadata:
+  name: peersample
```

Comment:
same here about the name. maybe make it just peer.

Reply:
fixing